iOS Client/PodcastsController/PodcastFilterTableViewCell.swift

```diff
@@ -0,0 +1,35 @@
+//
+//  PodcastFilterTableViewCell.swift
+//  iOS Client
+//
+//  Created by Jonathan Witten on 4/9/18.
+//  Copyright © 2018 Mockingbird. All rights reserved.
+//
+
+import UIKit
+
+class PodcastFilterTableViewCell: UITableViewCell {
+
+    @IBOutlet weak var toggleSwitch: UISwitch!
+    @IBOutlet weak var titleLabel: UILabel!
+    @IBOutlet weak var podcastImage: UIImageView!
+    
+    var podcast: PodcastStream?
+    
+    static var reuseIdentifier: String = "PodcastFilterTableViewCell"
+    func configure(image: UIImage? = nil, podcast: PodcastStream, on: Bool = false) {
+        toggleSwitch.isOn = on
+        titleLabel.text = podcast.title
+        podcastImage.image = image
+        self.podcast = podcast
+        podcastImage.layer.cornerRadius = 10
+        self.selectionStyle = .none
+    }
+    @IBAction func toggled(_ sender: UISwitch) {
+        guard let podcast = podcast else {
+            return
+        }
+        MBStore.sharedStore.dispatch(TogglePodcastFilter(podcastStream: podcast, visible: toggleSwitch.isOn))
```

Comment:
gotta love UDF

Reply:
Just a thought, but, if we're toggling something, and the state is tracking all of this stuff, then wouldn't the state know just to toggle it

`new_state = !old_state`? Why pass visible param?